### PR TITLE
updated to wp6.0.0

### PR DESCRIPTION
**Pull request by gin0115**
This PR contains an update for WP6.0.0

Have added 6.0 into `release-latest-versions.sh` not sure if that is needed or not 

> Thanks for creating and maintaining these stubs, makes my life as a wp dev so much better :)

### CODE REVIEW

File: release-latest-versions.sh [hunk 1]
```diff
@@ -35,7 +35,7 @@ CORE_JSON="$(wget -q -O- "https://packagist.org/packages/johnpbloch/wordpress-co
 
 # @TODO Use branches!
 for MINOR in                         4.7 4.8 4.9 \
-         5.0 5.1 5.2 5.3 5.4 5.5 5.6 5.7 5.8 5.9; do
+         5.0 5.1 5.2 5.3 5.4 5.5 5.6 5.7 5.8 5.9 6.0; do
```
Comment (szepeviktor) [inline release-latest-versions.sh:38]: Thank you for your PR.
I'm sorry to say that this is automatic repo (manually automatic).
But give this PR a try.

Please add 6.0 in a new line (like 5.0)

File: source/composer.json [hunk 2]
```diff
@@ -7,12 +7,12 @@
...
+        "johnpbloch/wordpress": "6.0.0"
     },
     "minimum-stability": "stable",
     "config": {
         "allow-plugins": {
             "johnpbloch/wordpress-core-installer": true
         }
     }
-}
+}
```
Comment (szepeviktor) [inline source/composer.json:18]: Please revert the last line end.

**Comment by szepeviktor** [comment]
@gin0115 What new files came with 6.0?

**Comment by gin0115** [comment]
Have made the changes suggested and yes it's mostly just block related, but nice to have all my pipelines running with the same version. 

Thanks for merging :)

**Comment by szepeviktor** [comment]
An advise for new software releases.

> Avoiding installation of new major version releases of any software in production, letting others test it first

**Pull request merged**